src/translate.py

```diff
@@ -1418,7 +1454,7 @@ def from_expr(expr: Expression) -> "SwitchControl":
                 offset = -offset_lit.value
 
         # Check that it is really a jump table
-        if not symbol_name_is_jump_table(jump_table.symbol_name):
+        if jump_table.asm_data_entry is None or not jump_table.asm_data_entry.is_jtbl:
```

Comment:
I just added this fn, but I think it's better to just use the `is_jtbl` attribute that's already there! :facepalm:

Reply:
heh, fair enough. Should probably inline the function at its only usage site again, so it's closer in sync with the error message